### ISSUE TITLE
table creation

### ISSUE THREAD

**Issue by konChristidis** [dimoskokkonos/AlbumCrud#5]
https://github.com/dimoskokkonos/AlbumCrud/blob/b542d98b4a8988cc1d836d6f88a5aa9d0ea29fd6/grails-app/controllers/crudmusicmk2/AlbumController.groovy#L18

Κάθε φορά που μπαίνεις στο index προσπαθείς να φτιάξεις από την αρχή τους πίνακες? Θέλω να μου εξηγήσεις πώς το σκέφτεσαι αυτό, γιατί φαίνεται περίεργο.

**Comment by dimoskokkonos** [comment]
Ναι, δεν είναι κάτι που πρέπει να είναι στο index. Στην αρχή δοκίμαζα τα δεδομένα και πως τα περνάω, αν είναι σωστά κλπ και επείδη έκανα πολλά drop και edit τους έβαλα να φτιάχνονται εκεί..  Έχει ξεμείνει απλά ακόμα στο index, θα το μετακινήσω σε άλλο action
